### ISSUE TITLE
WCAG 2.4.1: Page must have means to bypass repeated blocks (html)

### ISSUE THREAD

**Issue by SahilTara** [Microsoft/WebTemplateStudio#120]
**Issue**: `Page must have means to bypass repeated blocks` ([`bypass`](https://dequeuniversity.com/rules/axe/3.1/bypass?application=axeAPI))

**Target application**: [Project Acorn](http://localhost:3000/#/SelectWebApp)
**Happens with All endpoints**

**Element path**: html

**Snippet**:

    <html lang="en">

**How to fix**:

    Fix any of the following:
      No valid skip link found
      Page does not have a header
      Page does not have a landmark region

**Environment**:
Chrome version 72.0.3626.121

====

This accessibility issue was found using Accessibility Insights for Web 2.0.4 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
